presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -1397,6 +1397,30 @@ public void testChiSquaredCdf()
         assertInvalidFunction("chi_squared_cdf(-3, 0.3)", "df must be greater than 0");
         assertInvalidFunction("chi_squared_cdf(3, -10)", "value must non-negative");
     }
+        
+    @Test
+    public void testInversePoissonCdf()
+    {
+        assertFunction("inverse_poisson_cdf(3, 0.0)", DOUBLE, 0.0);
+        // assertFunction("inverse_poisson_cdf(3, 1.0)", DOUBLE, Double.POSITIVE_INFINITY);
```

Comment:
Why is this one commented out? Please fix. Thanks!

Reply:
removed.